src/Components/AdditionalRepositoriesTable/__tests__/AdditionalRepositoriesTable.test.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { fireEvent, render, screen } from '@testing-library/react';
+import AdditionalRepositoriesTable from '../AdditionalRepositoriesTable';
+
+jest.mock('../../../hooks/useFeatureFlag');
+import '@testing-library/jest-dom';
+
+const queryClient = new QueryClient();
+
+describe('AdditionalRepositoriesTable', () => {
+  const repositories = [
+    {
+      repositoryLabel: 'label-a',
+    },
+    {
+      repositoryLabel: 'label-b',
+    },
+  ];
+  it('renders correctly', () => {
+    const Table = () => (
+      <QueryClientProvider client={queryClient}>
+        <AdditionalRepositoriesTable repositories={repositories} />
+      </QueryClientProvider>
+    );
+
+    const { container } = render(<Table />);
+
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('when row column headings are clicked', () => {
+  const repositories = [
+    {
+      repositoryLabel: 'label-a',
+    },
+    {
+      repositoryLabel: 'label-b',
+    },
+    {
+      repositoryLabel: 'label-c',
+    },
+  ];
+  it('can sort by label', () => {
+    const Table = () => (
+      <QueryClientProvider client={queryClient}>
+        <AdditionalRepositoriesTable repositories={repositories} />
+      </QueryClientProvider>
+    );
+
+    const { container } = render(<Table />);
+    fireEvent.click(screen.getByText('Label'));
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('can sort by label, reversed', () => {
+    const Table = () => (
+      <QueryClientProvider client={queryClient}>
+        <AdditionalRepositoriesTable repositories={repositories} />
+      </QueryClientProvider>
+    );
+
+    const { container } = render(<Table />);
+    fireEvent.click(screen.getByText('Label'));
+    fireEvent.click(screen.getByText('Label'));
+
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('when using pagination', () => {
+  const repositories = [...Array(12)].map((id) => ({
+    repositoryLabel: `label-${id}`,
+  }));
```

Comment:
```suggestion
  const repositories = [...Array(12).keys()].map((id) => ({
    repositoryLabel: `label-${id}`,
  }));
```
You need `.keys()` here - if you look in the snapshot you'll see the labels are all undefined. The _keys_ get initialized with that array constructor but their _values_ do not